### PR TITLE
Porting gondola from tgstation

### PR DESCRIPTION
**Pull request by carshalash**
image

### CODE REVIEW

**Comment by jknpj** [comment]
>literally copy paste from github even forgot to remove the +
>does not modify the DME

You need to pratice your memefu

**Comment by TheFriggenSawbones** [comment]
the biggest mistake

File: vgstation13.dme [hunk 1]
```diff
@@ -1586,6 +1586,7 @@
 #include "code\modules\mob\living\simple_animal\parrot.dm"
 #include "code\modules\mob\living\simple_animal\puddi.dm"
 #include "code\modules\mob\living\simple_animal\roach.dm"
+#include "code/modules/mob/living/simple_animal/gondola.dm"
```
Comment (TheFriggenSawbones) [inline vgstation13.dme]: o\\\\\\\\\\\\\\\\\\\\\\\\\\\\o

**Review by TheFriggenSawbones — COMMENTED** [review]
**\\\\\\**

**Comment by DrSnips** [comment]
>hide is valuable
no hide from it

**Comment by DamianX** [comment]
repoban-tier pr

**Comment by carshalash** [comment]
Memed too hard and jumped the gun

**Comment by Exxion** [comment]
Why didn't I get notified about this PR until just now? I missed the opportunity to rain down righteous fury upon this garbage

**Comment by SonixApache** [comment]
RIP gondolas, we hardly knew ye

**Pull request closed without merging**